code/tests/control/carla_sim.py

```diff
@@ -161,7 +162,7 @@ def main(use_lane_detector=False, ex=False):
         # Create a synchronous mode context.
         with CarlaSyncMode(world, *sensors, fps=FPS) as sync_mode:
             while True:
-                if should_quit():

```

Comment:
Yeah, I think I should not have changed the name of this function. Sorry about that. But for some reason creating another function that again checks the pygame event did not work as expected.